src/MICore/LaunchOptions.cs

```diff
@@ -551,6 +551,15 @@ private static string ResolveFromPath(string command)
                 {
                     return candidate;
                 }
+
+                if (PlatformUtilities.IsWindows() && String.IsNullOrEmpty(Path.GetExtension(command)))
```

Comment:
>String.IsNullOrEmpty(Path.GetExtension(command))) [](start = 53, length = 49)

nit: hoist this outside of the loop so that a new string doesn't need to be allocated for each `pathPart`.

```
bool addExtension = PlatformUtilities.IsWindows() && String.IsNullOrEmpty(Path.GetExtension(command));
foreach (...
```